recipe/meta.yaml

```diff
@@ -36,21 +36,20 @@ requirements:
     - python
     - pip
     - pybind11
-    - pytorch
     - pytorch =*={{ torch_proc_type }}*
     - setuptools
     - pytest-runner
     - scipy
     - libxcrypt               # [linux and py<39]
     {% if cuda_major >= 12 %}
-    - cuda-driver-dev
-    - cuda-cudart-dev
-    - cuda-nvrtc-dev
-    - cuda-nvtx-dev
-    - cuda-nvml-dev
+    - cuda-driver-dev ={{ cuda_compiler_version }}*
+    - cuda-cudart-dev ={{ cuda_compiler_version }}*
+    - cuda-nvrtc-dev ={{ cuda_compiler_version }}*
+    - cuda-nvtx-dev ={{ cuda_compiler_version }}*
+    - cuda-nvml-dev ={{ cuda_compiler_version }}*
```

Comment:
There's an easier way to do this, just add a line with `cuda-version {{ cuda_compiler_version }}`, which will enforce the right set of packages.